docs/examples/FormInputAddons.js

```diff
@@ -0,0 +1,71 @@
+const formInstance = (
+  <form>
+    <FormGroup>
+      <InputGroup>
+        <InputGroup.Addon>@</InputGroup.Addon>
+        <FormControl type="text" />
+      </InputGroup>
+    </FormGroup>
+    <FormGroup>
+      <InputGroup>
+        <FormControl type="text" />
+        <InputGroup.Addon>.00</InputGroup.Addon>
+      </InputGroup>
+    </FormGroup>
+    <FormGroup>
+      <InputGroup>
+        <InputGroup.Addon>$</InputGroup.Addon>
+        <FormControl type="text" />
+        <InputGroup.Addon>.00</InputGroup.Addon>
+      </InputGroup>
+    </FormGroup>
+
+    <FormGroup>
+      <InputGroup>
+        <FormControl type="text" />
+        <InputGroup.Addon>
+          <Glyphicon glyph="music" />
+        </InputGroup.Addon>
+      </InputGroup>
+    </FormGroup>
+
+    <FormGroup>
+      <InputGroup>
+        <InputGroup.Button>
+          <Button>Before</Button>
+        </InputGroup.Button>
+        <FormControl type="text" />
+      </InputGroup>
+    </FormGroup>
+    <FormGroup>
+      <InputGroup>
+        <FormControl type="text" />
+        {/* Set z-index here to work around some layout quirks. */}
+        <InputGroup.Button style={{ zIndex: 1000 }}>
```

Comment:
the `zIndex` is unfortunate... is there a better API option here other then requiring these nested button groups? `InputGroup.Dropdown`, or sniff the dropdown child and avoid the nesting?